src/OpenSearch.Client/Cluster/ClusterStats/ClusterNodesStats.cs

```diff
@@ -235,6 +235,16 @@ public class ClusterOperatingSystemName
 		public string Name { get; internal set; }
 	}
 
+	[DataContract]
+	public class ClusterOperatingSystemPrettyName
```

Comment:
Please, update class def in `NodeInfo.cs`. You have duplicates now.

Reply:
Have removed the duplicate from `NodeInfo.cs`